tests/requirements-libraries.txt

```diff
@@ -18,7 +18,8 @@
 # - v. 2.2 and above fails.
 Django==2.1.8  # pyup: ignore
 # - v 3.1.0 fails.
-matplotlib==3.0.3  # pyup: ignore
+matplotlib==3.0.3; python_version < '3.8'  # pyup: ignore
+matplotlib==3.2.2; python_version >= '3.8'  # pyup: ignore
```

Comment:
The matplotlib hooks need updating so you'll have to change this soon. I hope I can get this done today.

Reply:
Ok after an unexpected work item came up I'm not going to get this done today. Hopefully in the next few days though!

Reply:
The changes address the `3.1.0` incompatibility, stemming from `rcParams`.